src/cli/remotes-wizard.js

```diff
@@ -8,21 +8,23 @@ export default async function (testCafe, remoteCount, showQRCode) {
 
     if (remoteCount) {
         log.hideSpinner();
-
+        if (remoteCount === 1)
+            log.write(`To connect a remote browser to TestCafe,\nnavigate to the following URL from that browser.`);
+	else
+            log.write(`${remoteCount} browsers are expected to connect.\nTo connect a remote browser to TestCafe,\nnavigate to the appropriate URL from that browser.`);
+        if (showQRCode)
+            log.write('Or scan the QR-code.');
```

Comment:
@inikulin, Do you like the Vasily's variant? Should I change the code according this variant?